proposal/color-4-new-spaces-js.changes.md

```diff
@@ -0,0 +1,21 @@
+## Draft 1.1
+
+* Clarify values in `channels` and `channelsOrNull`.
+
+* Throw an error if construction space can not be determined.
+
+* Remove `alpha` from list of deprecated getters.
+
+* Rename types: `ColorSpaceLAB` to `ColorSpaceLab`, `ChannelNameLAB` to
+  `ChannelNameLab`.
+
+* Use `Exclude<>` instead of `Omit<>` for union types.
+
+* Make procedure for determining space backwards compatible when using `change`
+  for legacy colors.
```

Comment:
I think this is no longer relevant.

Reply:
@nex3 I think this is still relevant for the [changes below](https://github.com/sass/sass/pull/3704/files#diff-c478516801a088f8c5bb0f50f2949f93259f54932decf8367c5d6a733ee6e17aR394-R400), updating how `space` is determined based on the presence of `whiteness`/`blackness`, `hue`/`saturation`/`lightness` etc.